Makefile

```diff
@@ -4,9 +4,6 @@ CURDIR := $(shell pwd)
 
 KEEP_FILE := '**/*.proto,**/*.sh'
 
-export PATH := $(CURDIR)/_vendor/bin:$(PATH)
-export GOPATH := $(CURDIR)/_vendor
```

Comment:
By removing these lines, generate_go may install few go tools into your real gopath and use wrong protobuf and grpc compiler.